jobs/pipeline-jobs.yml

```diff
@@ -51,6 +51,13 @@
     jobs:
       - 'pipeline-template':
 
+- project:
+    name: cru-udp-pipeline
+    description: |
+      Cru User Data Platform pipeline (serverless) function.
+    jobs:
+    - 'pipeline-template':
+        branches: master
```

Comment:
you don't want to have a staging env?

Reply:
We don't have a staging snowplow pipeline anymore.